subsys/ipc/rpmsg_service/rpmsg_backend.c

```diff
@@ -130,9 +130,8 @@ static void ipc_virtio_notify(struct virtqueue *vq)
 
 	status = ipm_send(ipm_handle, 0, current_core ? 0 : 1, 0, 1);
 #else
-	uint32_t dummy_data = 0x55005500; /* Some data must be provided */
-
-	status = ipm_send(ipm_handle, 0, 0, &dummy_data, sizeof(dummy_data));
+	/* No data transfer, only doorbell. */
+	status = ipm_send(ipm_handle, 0, 0, NULL, 0);
```

Comment:
Seems to me quite dangerous. Are we sure that all existing ipm drivers will support it without crash?

Reply:
Hmm nice catch @arnopo , some drivers may return error and the cross-core interrupt is not generated in case of empty data, of course we would receive an error here, but this change may cause breaks on other platforms. In that case I'm in favor of reverting this back.

Reply:
Resolved by adding a preprocessor guard.